app/bundles/ChannelBundle/Event/ChannelEvent.php

```diff
@@ -113,28 +112,4 @@ public function getFeatureChannels()
     {
         return $this->featureChannels;
     }
-
-    /**
-     * Set a preference center channel.
-     *
-     * @deprecated 2.4 to be removed 3.0; use addChannel()
-     *
-     * @param $channel
-     */
-    public function setChannel($channel)
```

Comment:
Removing these methods should be documented in https://docs.google.com/document/d/1lHNK8hHQtvyWlz9OtfAoLupDVb5-QunQjq-HAGlAsS4/edit#heading=h.imucd6w2ou47

Reply:
Good catch. I must have forgotten to do it.